deployment/helm/templates/udev.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.udev.enabled }}
+apiVersion: akri.sh/v0
+kind: Configuration
+metadata:
+  name: {{ .Values.udev.name }}
+spec:
+  protocol:
+    udev:
+      udevRules:
+      {{- toYaml .Values.udev.udevRules | nindent 6 }}
+  {{- if .Values.udev.brokerPod.image.repository }}
+  # only add broker pod spec if a broker image is provided
+  brokerPodSpec:
+    containers:
+    - name: {{ .Values.udev.name }}-broker
+      image: {{ .Values.udev.brokerPod.image.repository | quote }}
+      imagePullPolicy: {{ .Values.udev.brokerPod.image.pullPolicy }}
+      securityContext:
+        privileged: true
+      resources:
+        limits:
+          {{`"{{PLACEHOLDER}}"`}} : "1"
+    {{- with .Values.imagePullSecrets }}
+    imagePullSecrets:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+  {{- end }}
+  {{- if .Values.udev.brokerPod.image.repository }}
+  # only add service specs if a broker image was specified
+  instanceServiceSpec:
+    type: {{ .Values.udev.instanceService.type }}
+    ports:
+    - name: grpc
```

Comment:
Also wonder if for more complex scenarios, we should provide a sample config in the doc below to allow users to customize whether to have service specs or what the service specs should look like. Same for broker spec. Users might want to customize even the broker spec more than what we allow.

Reply:
Aaa, thanks. Perhaps just a link to that then. Is it possible for folks to drop instanceServiceSpec and  configurationServiceSpec if desired?

Reply:
Not currently. I can add that option.

Reply:
i like the idea of parameterizing the name ... i didn't for onvifVideo because our sample app was depending on the name.  but for this, i think it would be great .. .the default could be 'grpc`